src/clj_icat_direct/icat.clj

```diff
@@ -83,6 +83,12 @@
        (first)
        :path))
 
+(defn paths-for-uuids
+  [uuids]
+  (->> (q/mk-paths-for-uuids uuids)
+       (apply run-query-string)
+       (mapv :path)))
```

Comment:
I'm not sure how this would impact performance, but it seems like returning only the paths limits the usefulness of this, and it would be much more useful to return an array-of-arrays or a map between uuids and paths, such that this can be used for cases where the specific mappings matter.

Reply:
That's a good idea. I don't know why I didn't think of it.